packages/remix-dev/compiler/utils/tsconfig/write-config-defaults.ts

```diff
@@ -68,7 +71,11 @@ export function writeConfigDefaults(configPath: string) {
   let requiredChanges = [];
 
   if (!("include" in fullConfig)) {
-    config.include = ["remix.env.d.ts", "**/*.ts", "**/*.tsx"];
+    if (configType === "jsconfig.json") {
+      config.include = ["**/*.js", "**/*.jsx"];
+    } else {
+      config.include = ["remix.env.d.ts", "**/*.ts", "**/*.tsx"];
+    }
```

Comment:
Maybe make this a ternary?

```suggestion
    config.include = configType === "jsconfig.json"
      ? ["**/*.js", "**/*.jsx"]
      : ["remix.env.d.ts", "**/*.ts", "**/*.tsx"];
```